.github/workflows/getsentry.yml

```diff
@@ -0,0 +1,93 @@
+name: getsentry
+on: pull_request
+
+jobs:
+  ensure_docker:
+    name: Ensure docker image
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: getsentry token
+      id: getsentry
+      uses: getsentry/action-github-app-token@v1
+      with:
+        private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+    - name: Checkout getsentry
+      uses: actions/checkout@v2
+      with:
+        repository: getsentry/getsentry
+        token: ${{ steps.getsentry.outputs.token }}
+
+    - name: Install gcloud utils
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '281.0.0'
+        export_default_credentials: true
+        service_account_key: ${{ secrets.GCR_SERVICE_ACCOUNT }}
+
+    - name: Configure docker
+      run: |
+        mkdir -p $HOME/.docker
+        echo '{"experimental": "enabled"}' > $HOME/.docker/config.json
+        gcloud auth configure-docker --quiet
+
+    - name: bump-sentry ${{ github.event.pull_request.base.sha }}
+      run: |
+        git config --global user.email "github-actions@sentry.io"
+        git config --global user.name "github-actions[bot]"
+        bin/bump-sentry ${{ github.event.pull_request.head.sha }}
+
+    - name: Collect dependencies
+      run: bin/test-utils/collect_deps
+
+    - name: Ensure image
+      run: bin/test-utils/ensure_image
+
+  frontend:
+    name: frontend
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    needs: [ensure_docker]
+
+    steps:
+    - name: Checkout sentry
+      uses: actions/checkout@v2
+
+    - name: getsentry token
+      id: getsentry
+      uses: getsentry/action-github-app-token@v1
+      with:
+        private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+    - name: Checkout getsentry
+      uses: actions/checkout@v2
+      with:
+        repository: getsentry/getsentry
+        token: ${{ steps.getsentry.outputs.token }}
+
+    - name: Install gcloud utils
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        version: '281.0.0'
+        export_default_credentials: true
+        service_account_key: ${{ secrets.GCR_SERVICE_ACCOUNT }}
+
+    - name: Configure docker
+      run: |
+        mkdir -p $HOME/.docker
+        echo '{"experimental": "enabled"}' > $HOME/.docker/config.json
+        gcloud auth configure-docker --quiet
+
+    - name: bump-sentry ${{ github.event.pull_request.base.sha }}
+      run: |
+        git config --global user.email "github-actions@sentry.io"
+        git config --global user.name "github-actions[bot]"
+        bin/bump-sentry ${{ github.event.pull_request.head.sha }}
+
+    - name: Collect dependencies
+      run: bin/test-utils/collect_deps
+
+    - name: Run tests
+      run: |
+        bin/test-utils/docker_run_tests make ci-test-js
```

Comment:
Does this step run when an external contributor makes a pull request or does this job bail before cloning getsentry?

Reply:
Ah I left a `continue-on-error` which may have attempted to run this step, but it would have failed since `bin/test-utils` wouldn't exist since it's in getsentry.

Reply:
Removed in [`b72ce01` (#20167)](https://github.com/getsentry/sentry/pull/20167/commits/b72ce015e2818518de67f906dbba4cf77ef83d9f)

Reply:
So the `getsentry token` step is the one that should fail for forks?

Reply:
@markstory correct, it fails because it needs a github secret specific to this repo/org.